### ISSUE TITLE
British Isles Newspapers Unblocker

### ISSUE THREAD

**Issue by watsonbox** [donalmurtagh/irish-times-unblocker#1]
Do you have any plans to include some papers for your British pals? 🙏

**Comment by donalmurtagh** [comment]
Given the title of the repo, mainland UK and Channel Island newspapers are out of scope, but if you would like to unblock a Northern Irish newspaper, please submit a PR.
